src/components/ListShopping.js

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment, useState } from 'react';
+
+const ListShopping = () => {
+  const [name, setName] = useState();
+  const [list, setList] = useState([]);
+
+  const createItems = (event) => {
```

Comment:
very good, but you could use the already created class listproduct. 🥇

Reply:
Thanks for the tip @arche4 🙌